frontend/src/pages/pipelines/global/modelCustomization/ModelCustomization.tsx

```diff
@@ -0,0 +1,115 @@
+  const [ilabPipeline, setIlabPipeline] = React.useState<PipelineVersionKF | undefined>(undefined);
+
+  const [data, setData] = useGenericObjectState({
+    projectName: { value: !emptyProjects ? project.metadata.name : '' },
+  });
+
+  const setProject = (namespace: string) => {
+    setData('projectName', { value: namespace });
+    updateProject(namespace);
+  };
+
+  const validation = useValidation(data, modelCustomizationFormSchema);
+  const isDataDirty = React.useRef(data).current !== data;
+
+  React.useEffect(() => {
+    const findIlabPipeline = async () => {
+      let pipelineVersion: PipelineVersionKF | undefined;
+      setCanSubmit(!emptyProjects && isDataDirty && pipelinesServer.installed);
+      if (!emptyProjects) {
+        try {
+          const listPipelinesResponse = await api.listPipelines({});
+          const pipelinesArray = listPipelinesResponse.pipelines;
+          if (typeof pipelinesArray !== 'undefined') {
+            for (const pipeline of pipelinesArray) {
+              if (pipeline.display_name === ILAB_PIPELINE_NAME) {
+                const listPipelineVersionsResponse = await api.listPipelineVersions(
+                  {},
+                  pipeline.pipeline_id,
+                );
+                const pipelineVersionsArray = listPipelineVersionsResponse.pipeline_versions;
+                if (
+                  typeof pipelineVersionsArray !== 'undefined' &&
+                  pipelineVersionsArray.length > 0
+                ) {
+                  pipelineVersion = pipelineVersionsArray.at(0);
+                }
+              }
+            }
+          }
+        } catch (e) {
+          setIlabPipeline(pipelineVersion);
+        }
+      }
+      setIlabPipeline(pipelineVersion);
+    };
+
+    findIlabPipeline();
+  }, [api, emptyProjects, isDataDirty, pipelinesServer.installed]);
+
+  const isValid = validation.validationResult.success;
+
+  return (
+    <ValidationContext.Provider value={validation}>
+      <ApplicationsPage
+        title={modelCustomizationPageTitle}
+        description={modelCustomizationPageDescription}
+        breadcrumb={
+          <Breadcrumb>
+            <BreadcrumbItem to="/model-customization">Model customization</BreadcrumbItem>
+            <BreadcrumbItem>Instruct fine-tune run</BreadcrumbItem>
+          </Breadcrumb>
+        }
+        loaded
+        empty={false}
+      >
+        <EnsureAPIAvailability>
+          <PageSection hasBodyWrapper={false} isFilled>
+            <GenericSidebar
+              sections={Object.values(FineTunePageSections)}
+              titles={fineTunePageSectionTitles}
+              maxWidth={175}
+            >
+              <FineTunePage
+                updateProject={setProject}
+                isStartRunDisabled={
+                  (!canSubmit && !(typeof ilabPipeline !== 'undefined')) || !isValid
+                }
+                link={`/pipelines/${!emptyProjects ? project.metadata.name : ''}/${
```

Comment:
Use `encodeURIComponent` when using variables in link

```suggestion
                link={`/pipelines/${encodeURIComponent(!emptyProjects ? project.metadata.name : '')}/${
```